src/main/proto/cosmas.proto

```diff
@@ -0,0 +1,16 @@
+message Text {

```

Comment:
We use a naming convention where request protocol buffers are suffixed with `Request` and response protocol buffers are suffixed with `Response`, and their base name is normally the name of the method. If your method is `createNewVersion` then response protocol buffer is `CreateNewVersionResponse`. Please rename this and other messages appropriately.

Reply:
Please also add `syntax = "proto3";` line at the very top, because you are actually using proto3 synatx.